Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
isn't it because of the .editorconfig file? I also noticed a final blank line in the VERSION file.